lte/protos/pipelined.proto

```diff
@@ -87,6 +85,12 @@ message ActivateFlowsRequest {
 
   // APN AMBR info, passed at the time of session creation
   magma.lte.AggregatedMaximumBitrate apn_ambr = 6;
+
+  // Subscriber session ip address
+  IPAddress ip_addr = 7;
```

Comment:
if a session is created of pdn_type IPv4v6, then it can have both an IPv4 and IPv6 address. If it is IPv6, the 64-bit prefix part is of importance to identify the session as lower 64-bits for UE IP address can be changed during actual data communication.

Reply:
We're going to remove the `ip_addr` from the flow activation request soon, so I think this proto is fine for now.

Reply:
After that I'm not sure if we'll need something passed here, I don't fully understand IPv4v6 configuration

Reply:
As long as session is identified properly, there should not be any issues. If the session IP is not used, what will be used to identify the session, session ID?

Reply:
We've decided to do imsi + tunnel_id(uplink/downlink)